test/e2e/Tests/test.runsettings

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what is this file for? Is it for customers when they want to locally debug?

Reply:
Yep! By default, running the tests in the Visual Studio test explorer will test only Azure Storage. This file provides a way to configure the environment variables necessary for MSSQL testing locally